Package.resolved

```diff
@@ -24,8 +24,8 @@
         "repositoryURL": "https://github.com/getsentry/sentry-cocoa",
         "state": {
           "branch": null,
-          "revision": "dc00a3673e3d313e41326bd53802f757defd0eee",
-          "version": "7.28.0"
+          "revision": "4f6838af2688f8f762afdaea5bef941121e7d473",
+          "version": "8.0.0"
```

Comment:
Apologies for taking so long to review this PR. In the meantime, Sentry shipped a few updates. The latest version is [8.3.0](https://github.com/getsentry/sentry-cocoa/blob/main/CHANGELOG.md#830). Could we update it?